### ISSUE TITLE
If an IP Address is Bogon then isPublicIp is false

### ISSUE THREAD

**Issue by suryacyg** [johnnymastin/check-ip#4]
For **100.100.1.1**, the library is giving isPublicIp as false

`{
originalIp: '100.100.1.1',
boiledIp: '100.100.1.1',
isValid: true,
isBogon: true,
isApipa: false,
isMulticast: false,
isRfc1918: false,
isPublicIp: false
}`

I was assuming that isPublicIp: false because the IP is a bogon. If so, Is there a way to know whether an IP is public or not irrespective of whether the given input is bogon or not?

**Comment by johnnymastin** [comment]
Yes, that is correct.  isPublicIp is false because it isn't an IP that you can route to over the internet.  I believe this needs to be more clearly stated in the README.  

You would have to test multiple properties of the returned object to determine if an IP was public in the sense that bogons are public IPs but not routable.
